test/resolve/adapters/baseSourceAdapter.test.ts

```diff
@@ -81,18 +91,21 @@ describe('BaseSourceAdapter', () => {
 
   it('should resolve a folder component in metadata format', () => {
     const component = xmlInFolder.FOLDER_COMPONENT_MD_FORMAT;
+    assert(component.xml);
     const adapter = new DefaultSourceAdapter(component.type, undefined);
 
     expect(adapter.getComponent(component.xml)).to.deep.equal(component);
   });
 
-  it('should resolve a nested folder component in metadata format', () => {
+  it('should resolve a nested folder component in metadata format (document)', () => {
```

Comment:
the test was using the constants for report/folder but the type for Document.  It is 100% Document now.